### ISSUE TITLE
Take Action Landing wireframe

### ISSUE THREAD

**Issue by kristinsj** [radcampaign/defenders#42]
_[Current Take Action landing page](https://defenders.org/take-action/actions)_

- [ ] Title - Take Action
- [ ] Priority Action - big image (with photo credit) with prominent call to action. Sample text:

> **Save Alaska's Wolves and Bears From Cruel Killings** 
> The National Park Service has proposed lifting a ban on extreme killing methods on Park Service lands in Alaska, putting gray wolves and grizzly bears in jeopardy.
> _Speak Up >_

- [ ] Email / Mobile Sign-up
- [ ] Action List - Minus the featured action. Currently they have about 13 live actions. Can we fit all of them on the page? Or is paging required? Should be flexible to showing more or less. Include image, title, two sentence blurb.
- [ ] Activist Toolkit - This secondary page should be featured somehow, maybe an image and list of menu item/anchor links? General Tips, Communicate on Social Media, Attend a Town Hall, Write or Call Congress, Submit a Letter to the Editor, Face-to-Face Meetings with Elected Officials https://defenders.org/activist   
- [ ] Additional Links - 2-3 secondary links, for featuring other important content. In this case, I’d put:
  - Attend an Event - See what Defenders is up to in your area
  - Report Card - Find out where your Member of Congress stands 
  - Become a Defender - Join Defenders of Wildlife and stand up for North America’s wildlife and wild places.
- [ ] Success Stories - An automated view of blog posts tagged with “Success Stories.” Also add a link to, “See more Success Stories” to link users to that blog tag.

**Comment by kristinsj** [comment]
- [ ] Move Activist Hub links to the right of the list of actions. They want them higher on the page.
- [ ] Jennie suggested incorporating regional office links at the bottom, but in a different way than we currently have: "Could make this a scrolling set of buttons so it can stay as a thin horizontal section (though again, my concerns about too many block-shaped images)."

**Comment by xaqrox** [comment]
> Jennie suggested incorporating regional office links at the bottom, but in a different way than we currently have: "Could make this a scrolling set of buttons so it can stay as a thin horizontal section (though again, my concerns about too many block-shaped images)."
I'm not sure what this means... or if this a thing that made it past Brandy?

**Comment by kristinsj** [comment]
Basically they want a way to link to the regional offices, but not as giant as the 7 boxes section that already exists.

**Comment by jselt** [comment]
yeah I thought we were ignoring since it made no sense

**Comment by kristinsj** [comment]
I don't think it should be executed as described, but having a way to link to the field offices isn't a bad idea. I don't feel like this is part of the wild places/focal landscapes/regions discussion. We didn't specifically address this as a "don't do" in the meeting.

**Comment by jselt** [comment]
it seems like clutter on a take action page

**Comment by kristinsj** [comment]
I'll add a comment to Brandy then.

**Comment by jselt** [comment]
I mean I don't feel that strongly if you want it.

**Comment by kristinsj** [comment]
I mentioned adding it to #46 and #49 too, Staff and Event landing pages. Do regional office links make sense for either of those pages? I don't feel strongly about anything on a Friday afternoon.

**Comment by jselt** [comment]
I don't see why it would be useful on those either personally. If we do a staff page I assume a filter/grouping would be regional office and on an event node we'd list related staff, but not an event landing page - though again it might be filter on the events list.

**Comment by jselt** [comment]
On second thought maybe we should stick it in one page - so we have a design for it and can just have a view block with module styling that they can drop in where they want it.
